src/ResourceManager/Insights/Commands.Insights/Alerts/RemoveAzureRmAlertRuleCommand.cs

```diff
@@ -49,8 +50,8 @@ public class RemoveAzureRmAlertRuleCommand : ManagementCmdletBase
         /// </summary>
         protected override void ProcessRecordInternal()
         {
-            AzureOperationResponse result = this.InsightsManagementClient.AlertOperations.DeleteRuleAsync(resourceGroupName: this.ResourceGroup, ruleName: this.Name).Result;
-            WriteObject(result);
+            this.InsightsManagementClient.AlertRules.DeleteAsync(resourceGroupName: this.ResourceGroup, ruleName: this.Name).Wait();
+            WriteObject("");
```

Comment:
When I was writing the Swagger spec, I was told that "the rule" for delete operations is to return nothing.
Previously, the Hydra spec was returning the request Id and HttpStatus.